backend/index.js

```diff
@@ -1,11 +1,15 @@
 const express = require('express');
 const app = express();
-const axios = require('axios');
 require('dotenv').config();
 app.use(express.json())
 
 const port = 8080;
 
+const finnhub = require('finnhub');
+const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+api_key.apiKey = process.env.API_KEY;
```

Comment:
Can we name the key here something more descriptive. If we ever add more keys to the .env file then we need a good way to now which api key goes to which api.